easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.7a-foss-2020a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
```

Comment:
is there a reason not to use the mrbayes easyblock?

Reply:
MrBayes has fixed its build system and the easyblock is no longer needed (and it neither works with this version of the software)

Reply:
should a warning be added to the mrbayes easyblock in case someone else tries to use it?

Reply:
if the easyblock does not work with this version, we should indeed prevent/discourage its usage for it in one way or another.